routes/api/categories-api.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * All routes for category Data are defined here
+ * Since this file is loaded in server.js into api/categories,
+ *   these routes are mounted onto /api/categories
+ * See: https://expressjs.com/en/guide/using-middleware.html#middleware.router
+ */
+
+const { response } = require("express");
+const express = require("express");
+const router = express.Router();
+const categoryQueries = require('../../db/queries/categories');
+
+router.post('/add_category', (req, res) => {
```

Comment:
I'm not 100 percent sure about express...are we posting to api/ or api/add_category